.github/workflows/serverless-integration.yml

```diff
@@ -7,6 +7,8 @@ on:
       - 'pkg/serverless/**'
       - 'test/integration/serverless/**'
       - '.github/workflows/serverless-integration.yml'
```

Comment:
Hm I don't think this is related to your change, but shouldn't this PR have triggered our Serverless Integration Tests? 🤔

Reply:
That's odd, I originally had another commit with some unnecessary changes that I removed with a rebase and force push. Maybe that's why the integration tests weren't showing in the PR?

I just push a commit with a comment to `.github/workflows/serverless-integration.yml` and the integration tests started as expected